datumaro/components/annotation.py

```diff
@@ -207,6 +208,15 @@ class Label(Annotation):
     label: int = field(converter=int)
 
 
+@attrs(slots=True, order=False)
+class HashKey(Annotation):
+    _type = AnnotationType.hash_key
+    hash_key: List[str] = field(factory=list, validator=default_if_none(list))
```

Comment:
Can it be just `np.ndarray`?

Reply:
`hash_key` consists of list of string like `['eced...']`. So I set hash_key as `List[str]`.

Reply:
I set `hash_key: np.ndarray` and its shape is `(1,64)` with `np.uint8` due to using packbit.